### ISSUE TITLE
Set up Google Analytics

### ISSUE THREAD

**Issue by jrozra200** [CivicActions/datascience#15]

**Comment by lukefretwell** [comment]
@jrozra200 this is the analytics file of the contents that should be removed (using accessibility GA/Matomo):

https://github.com/CivicActions/datascience/blob/main/_includes/analytics.html

**Comment by lukefretwell** [comment]
@jrozra200 you're now an admin for the data science property.
